seed/tests/test_address_normalization.py

```diff
@@ -9,9 +9,9 @@
 from seed.utils.address import normalize_address_str
 
 
-class TestColumnListProfile(TestCase):
+class TestAddressNormalization(TestCase):
```

Comment:
not sure how this was `TestColumnListProfile` :(. I am definitely the one to blame for it... sorry!